src/m365/spfx/commands/project/project-doctor/doctor-1.20.0.ts

```diff
@@ -21,7 +21,7 @@ export default [
   new FN002015_DEVDEP_types_react('17'),
```

Comment:
Since you started updating `spfx project doctor` rules, update all rules to use object for args instead of separate variables.